src/daceInterface.ts

```diff
@@ -89,12 +90,31 @@ implements MessageReceiverInterface {
     private daemonRunning = false;
     private daemonBooting = false;
 
-    private port: number = vscode.workspace.getConfiguration(
-        'dace.interface'
-    ).port;
+    private port: number = -1;
+
+    private getRandomPort(callback: CallableFunction) {
+        const rangeMin = 1024;
+        const rangeMax = 65535;
+        const portCandidate = Math.floor(
+            Math.random() * (rangeMax - rangeMin) + rangeMin
+        );
+
+        const tempServer = net.createServer();
+        tempServer.listen(portCandidate, () => {
```

Comment:
why not do it with port 0?

Reply:
Getting the port after using that method is messy (string parsing), but we crucially need it.